### ISSUE TITLE
DIST_GIT_WEB_ROOT_URL option by itself has no effect

### ISSUE THREAD

**Issue by lubomir** [product-definition-center/product-definition-center#514]
If in settings_local you only define this one option, the value will be ignored. The reason is that DIST_GIT_REPO_FORMAT is the actual option that is read, and its value is constructed before the override comes into play.

Filed in Fedora Infra: https://pagure.io/fedora-infrastructure/issue/6538

**Comment by hluk** [comment]
Instead of:

```python
DIST_GIT_REPO_FORMAT = DIST_GIT_WEB_ROOT_URL + DIST_GIT_RPM_PATH + "%s"
```

it should be just:

```python
DIST_GIT_REPO_FORMAT = "%s"
```

But this could break configuration for someone.

Bigger issue is that options are not documented (should be listed in [docs](http://product-definition-center.github.io/product-definition-center/)).

**Comment by lubomir** [comment]
Fixed by #549
